pystarport/cluster.py

```diff
@@ -964,7 +964,7 @@ def relayer_chain_config_rly(data_dir, chain, relayer_chains_config):
     price = gas_price.get("price", 0)
     denom = gas_price.get("denom", "basecro")
     prices = f"{price}{denom}"
-    precompiled = chain.get("precompiled-contract-address", "")
+    precompiled = chain_config.get("precompiled_contract_address", "")
```

Comment:
and hermes use [precompiled_contract_address](https://github.com/devashishdxt/hermes/blob/1649b5d7ffa2102cc63f9d31bba482df6e631c06/crates/relayer/src/config.rs#L667)